internal/cmd/billing/command_cost_list.go

```diff
@@ -0,0 +1,114 @@
+package billing
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+type commandList struct {
+	*pcmd.AuthenticatedCLICommand
+}
+
+type costOut struct {
+	StartDate           string `human:"Start Date" serialized:"start_date"`
+	EndDate             string `human:"End Date" serialized:"end_date"`
+	Granularity         string `human:"Granularity" serialized:"granularity"`
+	LineType            string `human:"Line Type" serialized:"line_type"`
+	Product             string `human:"Product" serialized:"product"`
+	ResourceId          string `human:"Resource ID" serialized:"resource_id"`
+	ResourceDisplayName string `human:"Resource Display Name" serialized:"resource_display_name"`
+	EnvironmentId       string `human:"Environment ID" serialized:"environment_id"`
+	NetworkAccessType   string `human:"Network Access Type" serialized:"network_access_type"`
+	Price               string `human:"Price" serialized:"price"`
+	Unit                string `human:"Unit" serialized:"unit"`
+	OriginalAmount      string `human:"Original Amount" serialized:"original_amount"`
+	DiscountAmount      string `human:"Discount Amount" serialized:"discount_amount"`
+	Amount              string `human:"Amount" serialized:"amount"`
+}
+
+func (c *commandList) newCostListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "list <start-date> <end-date>",
```

Comment:
I'm thinking start date and end date should be required flags instead of arguments (for clarity).

Reply:
Thoughts on this @Jayxuchen? IMO, B is a bit more clear than A:

A:
```
confluent billing cost list 01-02-2003 01-03-2002
```

B:
```
confluent billing cost list --start-date 01-02-2003 --end-date 01-03-2002
```